app/experimenter/experiments/tests/test_tasks.py

```diff
@@ -373,3 +374,75 @@ def test_internal_only_does_not_add_comment(self):
         self.mock_bugzilla_requests_post.assert_not_called()
 
         self.assertEqual(Notification.objects.count(), 0)
+
+
+class TestUpdateExperimentStatus(
+    MockRequestMixin, MockNormandyMixin, TestCase
+):
+
+    def setUp(self):
+        super().setUp()
+
+        self.experiment = ExperimentFactory.create_with_status(
+            target_status=Experiment.STATUS_ACCEPTED, normandy_id=1234
+        )
+
+    def test_successful_experiment_status_update(self):
+        with MetricsMock() as mm:
+            tasks.update_experiment_status()
+            self.assertEqual(len(mm.get_records()), 4)
+            self.assertTrue(
+                mm.has_record(
+                    markus.INCR,
+                    "experiments.tasks.update_experiment_status.started",
+                    value=1,
+                )
+            )
+            self.assertTrue(
+                mm.has_record(
+                    markus.INCR,
+                    "experiments.tasks.update_experiment_status.updated",
+                    value=1,
+                )
+            )
+            self.assertTrue(
+                mm.has_record(
+                    markus.INCR,
+                    "experiments.tasks.update_experiment_status.completed",
+                    value=1,
+                )
+            )
+
+    def test_experiement_status_update_with_failure(self):
```

Comment:
It would probably be good if you made two mock experiments and force the normandy mocker to fail on one of them but not the other so we make sure if one fails the other will get updated.

You can set this up by making a little function that checks the id passed into the mock normandy call and if it's equal to one of the experiments, return a valid normandy response and if it's the other, throw the exception.